gtk4/Gir.toml

```diff
@@ -440,6 +440,9 @@ trust_return_value_nullability = false
     [[object.function]]
     name = "enumerate_printers"
     cfg_condition = "target_os = \"linux\""
+    [[object.function]]
+    name = "test_list_all_types"
+    manual = true
```

Comment:
Is this function actually useful without the whole `gtk_test` API?

Reply:
There are few missing testing API functions yes, but I plan to tackle those one after another :) some of them can't have simple bindings because they use some weird macros and have to be re-implemented manually.

Reply:
After looking at the missing test functions, there are only 4 of them. `test_init` is the most important one from the list, the three others are accessibility related and require a manual macro re-implementation for them to be useful. I will go ahead and merge this one for now and figure out `test_init` in the future as I can't really test it locally due to the default build of gtk4/glib on Fedora